src/practice.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+
+const style = {
+  arrow: {
+    'lineHeight': '280px'
```

Comment:
Absolute positioning might be more appropriate here. Or, at the very least percentage-based sizing/spacing instead of a hard-coded pixel value. Usually you only want hard-coded pixel values for stuff like our driving game or tiny art direction stuff like box shadows or borders.

Reply:
I switched the arrows to use absolute positioning. but i still hard coded the top, left, and right properties. Is this still a bad way to do this?